From 5cd67eefc7213cb6d9fc1d243af533182ac8ebc0 Mon Sep 17 00:00:00 2001
From: Kamil Debski <k.debski@samsung.com>
Date: Mon, 9 Sep 2013 16:52:59 +0200
Subject: [PATCH 0769/1302] tizen_defconfig update

Enable the new USB PHY driver using the Generic PHY Framework and enable
EHCI driver (it works well with new PHY driver).
Disable the old USB PHY driver.

Signed-off-by: Kamil Debski <k.debski@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/configs/tizen_defconfig | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/tizen_defconfig b/arch/arm/configs/tizen_defconfig
index 02ff16b..5062ba0 100644
--- a/arch/arm/configs/tizen_defconfig
+++ b/arch/arm/configs/tizen_defconfig
@@ -2341,7 +2341,7 @@ CONFIG_USB_DEFAULT_PERSIST=y
 CONFIG_USB_EHCI_HCD=y
 # CONFIG_USB_EHCI_ROOT_HUB_TT is not set
 CONFIG_USB_EHCI_TT_NEWSCHED=y
-# CONFIG_USB_EHCI_S5P is not set
+CONFIG_USB_EHCI_S5P=y
 # CONFIG_USB_EHCI_HCD_PLATFORM is not set
 # CONFIG_USB_OXU210HP_HCD is not set
 # CONFIG_USB_ISP116X_HCD is not set
@@ -2412,8 +2412,7 @@ CONFIG_USB_PHY=y
 # CONFIG_NOP_USB_XCEIV is not set
 # CONFIG_OMAP_CONTROL_USB is not set
 # CONFIG_OMAP_USB3 is not set
-CONFIG_SAMSUNG_USBPHY=y
-CONFIG_SAMSUNG_USB2PHY=y
+# CONFIG_SAMSUNG_USB2PHY is not set
 # CONFIG_SAMSUNG_USB3PHY is not set
 # CONFIG_USB_GPIO_VBUS is not set
 # CONFIG_USB_ISP1301 is not set
@@ -2964,8 +2963,15 @@ CONFIG_ARM_GIC=y
 CONFIG_GIC_NON_BANKED=y
 # CONFIG_IPACK_BUS is not set
 # CONFIG_RESET_CONTROLLER is not set
+
+#
+# PHY Subsystem
+#
 CONFIG_GENERIC_PHY=y
 CONFIG_PHY_EXYNOS_MIPI_VIDEO=y
+CONFIG_PHY_EXYNOS_USB=y
+CONFIG_PHY_EXYNOS4210_USB=y
+CONFIG_PHY_EXYNOS4212_USB=y
 
 #
 # File systems
-- 
1.8.3.2

